src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -107,8 +109,14 @@ ccjRequestedForDefendantClaimDetails %}
         {{ offerAcceptedForClaimantClaimDetails(claim) }}
       {% elif state.status === ClaimStatus.OFFER_SETTLEMENT_REACHED %}
         {{ offerSettlementReachedForClaimant(claim) }}
+      {% elif state.status === ClaimStatus.CLAIMANT_REJECTED_DEFENDANT_AS_COMPANY_OR_ORGANISATION_RESPONSE %}
+        {{ claimantRejectedDefendantAsCompanyOrOrganisationResponseDetailedStatus(claim) }}
       {% elif state.status === ClaimStatus.ELIGIBLE_FOR_CCJ_AFTER_BREACHED_SETTLEMENT %}
         {{ requestCCJ(claim) }}
+      {% else %}
```

Comment:
I removed this in my dashboard PR as there shouldn't be a catch all. it will mean people will potentially merge code that is broken without noticing

Reply:
Removed else block same as what you did in your dashboard bigger PR.